### ISSUE TITLE
Ligatures for != <= >= 

### ISSUE THREAD

**Issue by FuriouslyCurious** [i-tu/Hasklig#95]
I know Haskell doesn't need them but think of the other starving programming languages!  Your donation of only three ligatures will feed us for years.  YEARS!

**Comment by NoahAndrews** [comment]
Please!

**Comment by FuriouslyCurious** [comment]
Blackfont has some ligatures for C / Python: should we upstream them to Hasklig?

https://github.com/blackro-se/blackfont

**Comment by Krakob** [comment]
I'm not a Haskell user but I adore Source Code Pro and would really love if Hasklig were to be more of a general purpose font that caters to most languages.

**Comment by Asuza** [comment]
Please add these! But while you're in there, please don't forget *!==* and *===*.

**Comment by LanderlYoung** [comment]
I'm a fan of source code pro, and I use Java Kotlin.
Please add those symbols!

**Comment by vikky49** [comment]
@LanderlYoung @Asuza @Krakob @FuriouslyCurious I used [https://github.com/rojiani/Ligaturizer] to patch the Source code pro. It basically patches the all the FiraCode Ligatures onto any reasonable font. Its a great utility if u like both source code pro and the Fira code style ligatures. Try it out
<img width="1177" alt="scp ligatures" src="https://user-images.githubusercontent.com/180512/35290279-99c2d4e4-001e-11e8-8133-21bdb2f4aa88.png">

**Comment by LanderlYoung** [comment]
@vikky49 That's great!!!! I'll give it a try. (I was using blackfont previously)

In Chinese this is "授之以渔"！

![image](https://user-images.githubusercontent.com/5700847/111725778-ac327700-88a2-11eb-98bc-c8eceaf89502.png)


And here is the result. (I hope it doesn't have any copyright issue, if it does, please let me know, and I'll delete it.)

[SourceCodePro-Ligature.zip](https://github.com/i-tu/Hasklig/files/6168370/SourceCodePro-Ligature.zip)

**Comment by snaggen** [comment]
Ligaturizer is ok, but some ligatures like && doesn't really fit with the rest of the font...

**Comment by nino-moreton** [comment]
@snaggen you can exclude ligatures you don't want

**Comment by snaggen** [comment]
@nino-moreton yes, but I still like to have ligatures, just not ligatures that doesn't fit with the font. So I actually ended up doing just that and manually edit the font to add a && liggature I created myself. That is however not production quality, since I really don't know anything about editing fonts... but it works for me. So, I currently have a SourceCodePro font with ligatures that works for me... but it would be better to have a proper solution.

**Comment by PyvesB** [comment]
The author hasn't done much changes to the font himself in recent years but seems open to contributions. The only way to get this implemented is probably if someone submits a pull request.

**Comment by sgaliamov** [comment]
+1
